### PR TITLE
Insecure mode

### PR DESCRIPTION
**Pull request by jseldess**
Docs updates to account for recent change to `--insecure` flag. @petermattis, @mberhault, please take a look.

---
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/docs/177)

### CODE REVIEW

File: start-a-local-cluster.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Once you've [installed CockroachDB](install-cockroachdb.html), you can quickly s
     <button type="button" class="btn details collapsed" data-toggle="collapse" data-target="#details1">Details</button>
     <div id="details1" class="collapse">
       <ul>
-        <li> The <code>--insecure</code> flag sets client/server communication to insecure on the default port, 26257. To bind to different port, set <code>--port=&#60;port&#62;</code> and <code>--http-port=&#60;port&#62;</code>.</li>
+        <li> Client/server communication defaults to insecure, with the server listening only on <code>localhost</code> on port 26257. To bind to different port, set <code>--port=&#60;port&#62;</code> and <code>--http-port=&#60;port&#62;</code>.</li>
```
Comment (petermattis) [inline start-a-local-cluster.md]: You should mention that in order to listen (insecurely) on an external address the `--insecure` flag needs to be explicitly specified.

**Comment by petermattis** [comment]
LGTM

File: start-a-local-cluster.md (hunk 1 above)
Comment (jseldess) [inline start-a-local-cluster.md]: Fixed in last commit.

**Comment by jseldess** [comment]
Also found a way to use markdown within html (adding `markdown="1"` to the opening `<div> tag), so did that for the expand/collapse sections on start-a-local-cluster.md and secure-a-cluster.md

File: secure-a-cluster.md [hunk 2]
```diff
@@ -6,20 +6,21 @@ expand: true
...
-    <div id="details-secure1" class="collapse">
-        <ul>
-            <li>If you used the <code>cockroach start</code> commands on <a href="start-a-local-cluster.html">Start a Cluster</a> verbatim, the commands above will work as well. Otherwise, just set the <code>--http-port</code> flag to the ports you used.</li>
-            <li>For more details about the <code>cockroach quit</code> command, see <a href="stop-a-node.html">Stop a Node</a>.</li>
-        </ul>
+    <div id="details-secure1" class="collapse" markdown="1">
+
+    - If you used the `cockroach start` commands on [Start a Cluster](start-a-local-cluster.html) verbatim, the commands above will work as well. Otherwise, just set the `--http-port` flag to the ports you used.
+    - For more details about the `cockroach quit` command, see [Stop a Node](stop-a-node.html).
+    - If you leave the Admin UI open, when you restart the cluster with security (steps 3 and 4), you'll see "TLS handshake" errors until you adjust the URL to https (step 7).
```
Comment (petermattis) [inline secure-a-cluster.md]: The "TLS handshake" errors you see should be fixed by https://github.com/cockroachdb/cockroach/pull/5746.

Comment (jseldess) [inline secure-a-cluster.md]: Thanks, @petermattis. I'll update once that's merged.

**Pull request merged**